src/NServiceBus.AcceptanceTests/Recoverability/When_cross_q_transactional_message_is_moved_to_error_queue.cs

```diff
@@ -10,12 +11,10 @@
 
     public class When_cross_q_transactional_message_is_moved_to_error_queue : NServiceBusAcceptanceTest
     {
-        [TestCase(TransportTransactionMode.SendsAtomicWithReceive)]
-        public async Task Should_not_dispatch_outgoing_messages(TransportTransactionMode transactionMode)
+        [Test]
+        public async Task Should_not_dispatch_outgoing_messages()
         {
-            Requires.DtcSupport();
```

Comment:
You removed this, but didn't add `Requires.CrossQueueTransactionSupport()` like @timbussmann said it needed.